sys/include/net/sock/udp.h

```diff
@@ -94,11 +94,11 @@
  *     }
  * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
- * The application then waits indefinitely for an incoming message in
- * `buf` from `remote`. If we want to timeout this wait period we could
- * alternatively set the `timeout` parameter of @ref sock_udp_recv() to a
- * value `> 0`. If an error occurs on receive we just ignore it and continue
- * looping.
+ * The application then waits indefinitely for an incoming message in `buf`
+ * from `remote`. If we want to timeout this wait period we could alternatively
+ * set the `timeout` parameter of @ref sock_udp_recv() to a value != @ref
```

Comment:
Yeah, but together with the line change above the line would cross the 80 characters mark...